copy/copy.go

```diff
@@ -546,6 +551,25 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 		return nil, "", "", errors.Wrapf(err, "Error initializing image from source %s", transports.ImageName(c.rawSource.Reference()))
 	}
 
+	options.DestinationManifestCheck = true // TODO: Forced enablement for initial PR testing
+	if options.DestinationManifestCheck && options.SignBy == "" {
+		destImageSource, err := c.dest.Reference().NewImageSource(ctx, options.DestinationCtx)
+		if err == nil {
+			retManifest, retManifestType, err = destImageSource.GetManifest(ctx, nil)
+			if err == nil {
```

Comment:
Non-blocking: For the two error cases above, it would probably be useful to at least log them in the debug output.

Reply:
I've added these, but perhaps I should remove?

They will log if the destination image doesn't exist (albeit at debug level of course), which is a fairly common/expected scenario.